docs/collecting-container-logs.md

```diff
@@ -0,0 +1,170 @@
+      excludeContainerRegex: ""
+      ## A regular expression for hosts.
+      ## Matching hosts will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludeHostRegex: ""
+      ## A regular expression for namespaces.
+      ## Matching namespaces will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludeNamespaceRegex: ""
+      ## A regular expression for pods.
+      ## Matching pods will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
+      excludePodRegex: ""
+```
+
+For more advanced filtering logic, see [here](#advanced-filtering).
+
+### Modifying log records
+
+:construction: This needs `extraProcessors` for logs.
+
+### Persistence
+
+By default, the metadata enrichment service provisions and uses a Kubernetes PersistentVolume as an on-disk queue that guarantees
+durability across Pod restarts and buffering in case of exporting problems.
+
+This feature is enabled by default, but it only works if you have a correctly configured default `storageClass` in your cluster. Cloud
+providers will do this for you when provisioning the cluster. The only alternative is disabling persistence altogether.
+
+Persistence can be customized via the `metadata.logs.persistence` section:
+
+```yaml
+metadata:
+  persistence:
+    enabled: true
+    # storageClass: ""
+    accessMode: ReadWriteOnce
+    size: 10Gi
+    ## Add custom labels to all otelcol statefulset PVC (logs and metrics)
+    pvcLabels: {}
+```
+
+Note that these settings affect persistence for metrics as well.
+
+## Advanced Configuration
+
+There are two ways of directly configuring OpenTelemetry Collector for both log collection and metadata enrichment.
+These are both advanced features requiring a good understanding of this chart's architecture and
+OpenTelemetry Collector configuration.
+
+The `metadata.logs.config.merge` and `otellogs.config.merge` keys can be used to provide configuration that will be merged
+with the Helm Chart's default configuration. It should be noted that this field is not subject to
+normal backwards compatibility guarantees, the default configuration can change even in minor
+versions while preserving the same end-to-end behaviour. Use of this field is discouraged - ideally
+the necessary customizations should be able to be achieved without touching the otel configuration
+directly. Please open an issue if your use case requires the use of this field.
+
+The `metadata.logs.config.override` and `otellogs.config.override` keys can be used to provide configuration that will be completely
+replace the default configuration. As above, care must be taken not to depend on implementation details
+that may change between minor releases of this Chart.
+
+See [Sumologic OpenTelemetry Collector configuration][configuration] for more information.
+
+### Advanced Filtering
```

Comment:
I don't feel that `Advanced Filtering` should be subsection, if we mention only override and merge options

Reply:
I'm not sure I follow. How else would you like it to be structured?

Reply:
maybe putting it into another subsection `### Direct configuration changes` or sth like that

Reply:
So you'd like the following:
- Advanced configuration
  - Direct configuration
  - Advanced filtering
?

Reply:
Yes, exactly

Reply:
Sure, I can do that.

Reply:
Sure, I can do that.

Reply:
Done